<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JSONPropertyWidget</class>
 <widget class="QWidget" name="JSONPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JSONPropertyWidget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typeBox">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Text</string>
      </property>
      <property name="modelColumn">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="textEditorPAge">
       <widget class="QLineEdit" name="valueEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>6</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="boolEditorPage">
       <widget class="QCheckBox" name="boolValue">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="numberEditPage">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>201</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="nullPage">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>null</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
